templates/blog/post_list.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% for post in post_list %}
+    <div class="post">
+      <p><b>Title</b> : <a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></p>
+      <p>{{ post.owner }} at {{ post.created_time }}</p>
+      <p><b>Content</b> : {{ post.content }}</p>
+      <p>{{ post.commentaries.all.count }} Comment{{ post.commentaries.all.count|pluralize }}</p>
```

Comment:
The template correctly uses the `pluralize` filter to handle singular and plural forms for the number of comments. However, the method `post.commentaries.all.count` should be `post.commentaries.count` to avoid unnecessary loading of all commentaries into memory, which is more efficient.